Makefile

```
@@ -72,7 +72,7 @@ stop: ## Stop docker development environment
 	docker-compose stop
 
 config/config.yaml:
-	cp config/config.dev.yaml config/config.yaml
+	cat config/config.dev.yaml | sed "s/uuid: \"\"/uuid: \"$$USER.local\"/" > config/config.yaml
```

```suggestion
	cat config/config.dev.yaml | sed "s/uuid: \"\"/uuid: \"$$RANDOM.$$USER.local\"/" > config/config.yaml
```

keep it unique even if usernames aren't